inst/templates/wercker.yml

```diff
@@ -0,0 +1,5 @@
+box: rocker/hadleyverse
```

Comment:
Also @krlmlr I wrote a blog post about this setup if you haven't seen it, http://jimhester.com/post/wercker-and-rocker-finally-performant-continuous-integration-for-r.  Your post and repositories were part of my inspiration for working on it, so thanks for blazing the trail!